Documentation/CallGuide.md

```diff
@@ -113,13 +113,17 @@ peerConnection.addEventListener( 'signalingstatechange', event => {
 	};
 } );
 
-peerConnection.addEventListener( 'addstream', event => {
+peerConnection.addEventListener( 'track', event => {
 	// Grab the remote stream from the connected participant.
-	remoteMediaStream = event.stream;
+	const connection = event.target;
+	const stream = connection._remoteStreams.values().next().value;
+	remoteMediaStream = new MediaStream( stream );
```

Comment:
This doesn't do anything, you can just use the remote stream.

Reply:
Got it.